x11-packages/yad/build.sh

```diff
@@ -0,0 +1,30 @@
+TERMUX_PKG_HOMEPAGE=http://sourceforge.net/projects/yad-dialog/
+TERMUX_PKG_DESCRIPTION="Display graphical dialogs from shell scripts or command line"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=9.3
+TERMUX_PKG_SRCURL=https://github.com/v1cont/yad/releases/download/v${TERMUX_PKG_VERSION}/yad-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=d5ca05d7658ac45490f1b49e15d24acd2c2011d88dab3f8dab0431ae9f493319
+TERMUX_PKG_DEPENDS="gtk3, webkit2gtk-4.1, gtksourceview3, gspell"
+TERMUX_PKG_BUILD_DEPENDS="libxml2-utils"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	export LDFLAGS="${LDFLAGS} -landroid-shmem"
```

Comment:
You can just append to it.
```suggestion
	export LDFLAGS+=" -landroid-shmem"
```